pkgs/desktops/gnome-3/3.20/core/gtksourceviewmm/src.nix

```diff
@@ -0,0 +1,11 @@
+# Autogenerated by maintainers/scripts/gnome.sh update
+
+fetchurl: {
+  name = "gtksourceviewmm-3.18.0";
```

Comment:
No. There is a  3.21.2 version but it's a development version (https://developer.gnome.org/gtksourceviewmm/).